docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
     build: .
     env_file: ./.env
     ports:
-      - "$SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT"
+      - "8089:$SPRING_DOCKER_PORT"
```

Comment:
same

Reply:
it was done due to error, which I got when I want to run my application. Error was due to PID, so, when I killed this PID, I forgot to change back to holders.